### PR TITLE
Profile Update 

### PR DESCRIPTION
**Pull request by kshitij9896**

### CODE REVIEW

File: data/kshitij9896.json [hunk 1]
```diff
@@ -2,12 +2,33 @@
   "name": "Kshitij Bansod",
   "type": "personal",
   "bio": "Full Stack Web Developer | Designer | Author ",
-
+  "tags": [
+    "Developer",
+    "Front-End Developer",
+    "UI/UX",
```
Comment (swagataroy30) [inline data/kshitij9896.json:8]:
```suggestion
    "UI/UX"
```

**Review by swagataroy30 — COMMENTED** [review]
You need to remove the comma from line 8. Thank You!

**Review by Hrishi5111998 — APPROVED** [review]

File: data/kshitij9896.json (hunk 1 above)
Comment (shubhsharma19) [reply data/kshitij9896.json:8]: Looks good :)

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**